### PR TITLE
<Accordion/> - add bottom shadow to open item

### PR DESCRIPTION
**Pull request by mjurfest**
### 🔦 Summary

- Change accordion item shadow to the inner item
- Add bottom shadow to the inner item

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @EgoziE 
- 📚 Change is documented
  - [x] Story

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6509

File: src/Accordion/AccordionItem/AccordionItem.st.css [hunk 1]
```diff
@@ -68,6 +56,20 @@
   background-color: value(B60);
 }
 
+.root:open .children {
+  box-shadow: 
+    inset 0px 18px 8px -10px rgba(22, 45, 61, 0.05), 
+    inset 0px -15px 8px -10px rgba(22, 45, 61, 0.05);
```
Comment (EgoziE) [inline src/Accordion/AccordionItem/AccordionItem.st.css:62]: `rgba(22, 45, 61, 0.05)` is not a valid color for a shadow.

Comment (EgoziE) [reply src/Accordion/AccordionItem/AccordionItem.st.css:62]: Ask Kfir to talk with Ayelet

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/Accordion/AccordionItem/AccordionItem.st.css (hunk 1 above)
Comment (milkyfruit) [reply src/Accordion/AccordionItem/AccordionItem.st.css:62]: I played around a bit how to recreate the same style as Shadow30, but as inner: https://codepen.io/milkyfruit/pen/MWjXrVB

We should introduce new variables:
Inner shadow 30 top:
```
inset 0  36px  18px -18px rgba(22,45,61,0.12),
inset 0  12px   6px  -6px rgba(22,45,61,0.06)
```
Inner shadow 30 bottom:
```
inset 0 -36px  18px -18px rgba(22,45,61,0.12),
inset 0 -12px   6px  -6px rgba(22,45,61,0.06)
```

**Review by milkyfruit — CHANGES_REQUESTED** [review]

**Pull request merged**